tests/test_yaml_file_format.py

```diff
@@ -15,34 +15,34 @@ def test_file_format():
             {
                 "aliases": ["cgap-core:FileFormat-bam"],
                 "description": "format to represent aligned reads",
-                "extrafile_formats": ["bai"],
-                "file_format": "bam",
-                "institution": "/institutions/hms-dbmi/",
-                "project": "/projects/cgap-core/",
+                "extra_file_formats": ["bai"],
+                "identifier": "bam",
+                "submission_centers": ["hms-dbmi"],
+                "consortia": ["cgap-core"],
```

Comment:
Replace `cgap-core` with `smaht-consortia` where relevant, needed in many places

Reply:
oh this is just for the test as a random string, it does not interact with the portal at all. It could be any string